modules/op-configuring-buildah-to-use-build-user.adoc

```diff
@@ -10,11 +10,11 @@ You can define a Buildah task to use the `build` user with user id `1000`.
 
 .Procedure
 
-. Create a copy of the `buildah` cluster task as an ordinary task.
+. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace, renamed as `buildah-as-user`.
```

Comment:
Is it the openshift-pipelines namespace that is renamed as `buildah-as-user` or do they need to rename the copy of the buildah task to that name? If it's the latter, then I would slightly rewrite to be more clear.